source/geod24/concurrency.d

```diff
@@ -824,6 +824,9 @@ struct ThreadInfo
     Tid ident;
     Tid owner;
 
+    /// Storage of information required for scheduling, message passing, etc.
+    public Object[string] objects;
```

Comment:
I am not sure why this is here. This can just be `FiberScheduler scheduler;`, and `thisScheduler` could return it. Why is it a map, and why does it store `Object` instead of `FiberScheduler`?

It is not used to store any other types of objects in this PR.

Reply:
All Commitments can be found at PR #25.
It will be used later. What should I do in this case?

https://github.com/Geod24/localrest/pull/25/commits/297220c01b9530b3b12513422d1b05b7f1781222#diff-b2ce8db5a10ee88e89a581259145e669R115-R147

Reply:
Alright, so you want to store multiple types into `ThreadInfo`. But why not just store:

```
struct ThreadInfo
{
    FiberScheduler scheduler;
    MessageChannel channel;
}
```

And then add more things as it's necessary? It doesn't have to use a hashmap and be dynamic.

Reply:
Ok Michael explained it to me IRL, so the issue is that `ThreadInfo` is in `geod24.concurrency`, but the real type we want to store there is `Channel!Type`, and we don't know the type yet.

There might be a better way to do this, but I'll have to think about it. In the meantime, we could go with Michael's approche.

Reply:
Thank you for your review.

I made the following corrections.
struct ThreadInfo
{
    FiberScheduler scheduler;
}
If I need an Object later, I will add a full explanation there.